cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -40,22 +52,35 @@ export default {
     const dateRegEx = /\d{1,2}\/\d{1,2}\/\d{4},\s\d{1,2}:\d{2}\s\w{2}/gm;
 
     return cy.get('#circulation-log-list').then(element => {
-      const resultCount = element.attr('data-total-count');
+      let resultCount = element.attr('data-total-count');
```

Comment:
it can be const  and 1 line with element.attr('data-total-count')>29?29:element.attr('data-total-count')